kgforge/specializations/models/rdf/store_service.py

```diff
@@ -28,37 +31,40 @@
 
 class StoreService(RdfService):
 
-    def __init__(self, default_store: Store, context_iri: Optional[str] = None,
-                 context_store: Optional[Store] = None) -> None:
+    def __init__(
+        self,
+        default_store: Store,
+        context_iri: Optional[str] = None,
+        context_store: Optional[Store] = None,
+    ) -> None:
 
         self.default_store = default_store
         self.context_store = context_store or default_store
         # FIXME: define a store independent strategy
-        self.NXV = Namespace(self.default_store.service.namespace) if hasattr(self.default_store.service, "namespace") \
-            else Namespace(Service.NEXUS_NAMESPACE_FALLBACK)
-        self.store_metadata_iri = self.default_store.service.store_context if hasattr(self.default_store.service, "store_context") \
+        self.store_metadata_iri = (
+            self.default_store.service.store_context
+            if hasattr(self.default_store.service, "store_context")
             else Namespace(Service.NEXUS_CONTEXT_FALLBACK)
-        self._shapes_to_resources: Dict
-        self._imported = []
-        self._graph = Graph()
+        )
+        self._graph = rdflib.Dataset()
         self._sg = ShapesGraphWrapper(self._graph)
```

Comment:
as self._sg is used within RdfService, probably it should be initialized there

Reply:
The RdfService delegates the building of _sg to specific type of services.

Reply:
self._sg init is now moved to RDFService